src/core/error.extern.js

```diff
@@ -19,5 +19,10 @@
  * @externs
  */
 
+Error.prototype.expected;
+Error.prototype.never;
+Error.prototype.fromAssert;
```

Comment:
Some of these properties no longer exist. I'm only transferring externs as they become needed during typechecking, to avoid importing stale or mistaken ones

Reply:
removed this file